src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/Map.java

```diff
@@ -25,7 +26,7 @@ public Map(ConnectionImpl connection) {
 	@Override
 	public RecyclableCollection<ClientBoundPacketData> toData() {
 		ClientBoundPacketData serializer = ClientBoundPacketData.create(PEPacketIDs.MAP_ITEM_DATA);
-		VarNumberSerializer.writeSVarLong(serializer, itemData);
+		VarNumberSerializer.writeSVarInt(serializer, itemData);
 
```

Comment:
Is this a bug fix, or a change between 1.9/1.10 and 1.11?

Reply:
steadfast has a signed var int here, so just switched it over. we only have integer map data anyways, so shouldnt really matter, but this is more correct i think. (can confirm in ghidra once its done doing its migration)